run.py

```diff
@@ -1,15 +1,17 @@
+import logging
+
 from aiogram import types, F, Router
 from aiogram.filters import Command
 from aiogram.utils.keyboard import InlineKeyboardBuilder
 
 import config
-from config import SUPPORT_LINK
-import logging
 from bot import dp, main
-from multibot import main as main_multibot
+from config import SUPPORT_LINK
 from handlers.admin.admin import admin_router
 from handlers.user.all_categories import all_categories_router
+from handlers.user.cart import cart_router
 from handlers.user.my_profile import my_profile_router
+from multibot import main as main_multibot
```

Comment:
Why do you move lines up and down unnecessarily?

Reply:
as far as I can tell this was done by my IDE. okay for now to leave it?